api/src/main/java/org/pmiops/workbench/api/CloudTaskWorkspacesController.java

```diff
@@ -19,10 +20,14 @@ public class CloudTaskWorkspacesController implements CloudTaskWorkspacesApiDele
   private static final boolean DELETE_BILLING_PROJECTS = true;
 
   private final ImpersonatedWorkspaceService impersonatedWorkspaceService;
+  private final WorkspaceService workspaceService;
 
   @Autowired
-  public CloudTaskWorkspacesController(ImpersonatedWorkspaceService impersonatedWorkspaceService) {
+  public CloudTaskWorkspacesController(
+      ImpersonatedWorkspaceService impersonatedWorkspaceService,
```

Comment:
It did not seem like the deleteWorkspace call needs the elevated access in the ImpersonatedWorkspaceService.
It seems a little silly to have two WorkspaceServices in a single controller. Does anyone have suggestions here?